test/performance/odp_ipsec.c

```diff
@@ -643,41 +655,58 @@ run_measure_one(ipsec_args_t *cargs,
 
 	while ((packets_sent < cargs->iteration_count) ||
 	       (packets_received < cargs->iteration_count)) {
-		if ((packets_sent < cargs->iteration_count) &&
+		int num_out = cargs->iteration_count - packets_sent;
```

Comment:
Looking at the (packets_sent < cargs->iteration_count) condition, now the iteration_count is really packet count and not the number of iterations or number of times the IPsec enqueue API gets called. Maybe the description of -i in usage() should be updated?

Reply:
Agreed. Do you suggest renaming the variable as well? (like, iteration_count -> packet_count?)

Or would updating description of -i option good enough?

Reply:
Its probably good to rename the variable too for clarity.